pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

```diff
@@ -55,7 +55,7 @@ If you need to run Spark jobs on Kubernetes using Spark Operator, you must insta
 
 ## Add Kaptain to your Kommander Install
 
-The folloing catalog will need to be added to the Komamnder install configuration file alonig with the DKP applications catalog:
+The following catalog will need to be added to the Kommander install configuration file along with the DKP applications catalog:
```

Comment:
Just a nit, but I think this would scan better in active voice:

```suggestion
Add the Kaptain catalog to the Kommander install configuration file along with the DKP applications catalog:
```